opencga-app/src/main/java/org/opencb/opencga/app/migrations/v3_0_0/OrganizationMigration.java

```diff
@@ -377,17 +377,22 @@ protected void run() throws Exception {
                 if (configuration.getAuthentication() != null && CollectionUtils.isNotEmpty(configuration.getAuthentication().getAuthenticationOrigins())) {
                     for (AuthenticationOrigin authenticationOrigin : configuration.getAuthentication().getAuthenticationOrigins()) {
                         if (authenticationOrigin.getType().equals(AuthenticationOrigin.AuthenticationType.OPENCGA)
-                                && authenticationOrigin.getId().equals(CatalogAuthenticationManager.INTERNAL)) {
+                                && "internal".equals(authenticationOrigin.getId())) {
                             continue;
                         }
-                        authenticationOrigin.setAlgorithm(algorithm);
-                        authenticationOrigin.setSecretKey(secretKey);
-                        authenticationOrigin.setExpiration(3600);
-                        authOrigins.add(convertToDocument(authenticationOrigin));
+                        Document authOriginDoc = convertToDocument(authenticationOrigin);
```

Comment:
As this super critical migration has changed, should we explicitely test this migration again?

Reply:
Yes, we should